### PR TITLE
SS Whitehead

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/02-journal.adoc [hunk 1]
```diff
@@ -4444,8 +4484,11 @@ Robert Stebbing, Thomas Grigg, Charles Wright, John Gray, John Jolley, Samuel Hu
 Edward Keen, Richard Humphrey, Joseph Haymor, Richard Clopton, Peter Cole, Robert Grows,
 Edmund Darby, Thomas Brook, Richard Thurston, William Goodale.
 
-[.asterism]
-'''
+--
+//not sure if this should be embedded or not.^^^
+
+[.embedded-content-document.letter]
```
Comment (maijapapaya) [inline journal/modernized/02-journal.adoc]: Should this abstract be labeled a letter? Or even embedded at all?

File: journal/modernized/02-journal.adoc [hunk 2]
```diff
@@ -1268,6 +1268,9 @@ so that the Lord delivered us by removing our unmerciful oppressor.
 [.asterism]
 '''
 
+[.embedded-content-document.address]
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc:1268]: Let's nix the asterism above

File: journal/modernized/02-journal.adoc [hunk 3]
```diff
@@ -2337,6 +2342,9 @@ A poor pretense for detaining me fifteen months in prison!
 [.asterism]
 '''
 
+[.embedded-content-document.letter]
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc:2339]: also, nix asterism above

File: journal/modernized/02-journal.adoc [hunk 4]
```diff
@@ -2412,12 +2420,17 @@ George Whitehead.
...
 
+--
+
 This letter Thomas Waldgrave produced in court at the aforesaid quarter sessions,
 and the only exception he made against it was, my giving him the title of T. Waldgrave.
 I told him I followed his own subscription to his mittimus, whereby he committed me,
 having therein so subscribed himself, T. Waldgrave; but he concluded,
 it was not for me to do as he had done; whereby he proved no crime at all against me.
 
+[.embedded-content-document.legal]
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc:2425]: nice job on the `.legal` 👍

File: journal/modernized/02-journal.adoc [hunk 5]
```diff
@@ -2431,6 +2444,8 @@ to be holden for this county at Bury,
 then and there to be proceeded withal according to law;
 and hereof I require you not to fail; this 1st day of August, 1655.`"
 
+--
+
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc:2442]: let's remove the small-break afterwards

File: journal/modernized/02-journal.adoc [hunk 6]
```diff
@@ -2496,6 +2516,9 @@ How evident is it therefore, that their own invidious wills were their only laws
 [.small-break]
 '''
 
+[.embedded-content-document.letter]
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc:2507]: I think we should kill the small-break above here too.

File: journal/modernized/02-journal.adoc [hunk 7]
```diff
@@ -4444,8 +4484,11 @@ Robert Stebbing, Thomas Grigg, Charles Wright, John Gray, John Jolley, Samuel Hu
 Edward Keen, Richard Humphrey, Joseph Haymor, Richard Clopton, Peter Cole, Robert Grows,
 Edmund Darby, Thomas Brook, Richard Thurston, William Goodale.
 
-[.asterism]
-'''
+--
+//not sure if this should be embedded or not.^^^
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc]: I think you did it right. 👍

File: journal/modernized/02-journal.adoc (hunk 1 above)
Comment (jaredh159) [reply journal/modernized/02-journal.adoc]: Yes, I think it should be embedded, but let's change it to `address`

File: journal/modernized/02-journal.adoc [hunk 8]
```diff
@@ -4533,9 +4576,14 @@ were sworn against us.`'`"
 Thus I have abstracted the substance and greatest part of our original answer,
 which was signed by Robert Duncon, George Whitehead, and Robert Grassingham.
 
+--
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc]: I think we need to move this closing `--` up a few lines to _before_ `Thus I have abstracted...`

File: journal/modernized/02-journal.adoc [hunk 9]
```diff
@@ -9365,6 +9447,8 @@ to prevent and hinder such meetings at their peril.
 [.signed-section-signature]
 John Nicholas.
 
+--
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc:9437]: Whoa... Something is super messed up here (not your fault -- a problem in the existing document).  It looks like the whole chunk of this embedded document is missing.  See below image.  Ignore for now, when you submit your updates, I'll add the missing chunk back in.  Leaving this as a note for myself.

missing

File: journal/modernized/02-journal.adoc [hunk 10]
```diff
@@ -9777,6 +9864,8 @@ for which this shall be your warrant.
 Given at our court at Whitehall, the -- day of June,
 in the twenty-fourth year of our reign.
 To our attorney general.
+//Does there need to be a closing above?
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc]: No, I don't think so, you can leave it as is.

File: journal/modernized/02-journal.adoc [hunk 11]
```diff
@@ -9884,6 +9976,8 @@ Arlington.
 [.signed-section-context-close]
 At the court at Whitehall, the 13th of Sept, 1672.
 
+--
```
Comment (jaredh159) [inline journal/modernized/02-journal.adoc:9966]: whoah, another one.  I'll fix this one too.  the actual content is missing....

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Great work as always.  Left a few comments for you to change a few things, and I left some comments for myself because I found some weird missing chunks.  I'll fix those after you have your stuff corrected, ignore those comments. Thanks!

**Comment by jaredh159** [comment]
Nice job, thanks! Merged.

**Pull request merged**